packages/eslint-config-nimble-core/package.json

```
@@ -30,7 +27,7 @@
     "lib": "lib"
   },
   "engines": {
-    "node": "^14.18.2 || ^16 || ^17 || ^18"
+    "node": ">=18"
```

Since one of our client projects (easyHotel) still stays in node 14 :harold:, please change it to
```suggestion
    "node": "^14.18.2 || >=18"
```
or 
```suggestion
    "node": "^14.18.2 || >=16"
```
and others as well

---

It was updated it in 0d8401f ✅